packages/agreement-process/src/routers/AgreementRouter.ts

```diff
@@ -12,8 +12,13 @@ import {
   agreementToApiAgreement,
   apiAgreementStateToAgreementState,
 } from "../model/domain/apiConverter.js";
-import { agreementService } from "../services/agreementService.js";
+import { config } from "../utilities/config.js";
 import { agreementNotFound } from "../model/domain/errors.js";
+import { agreementServiceBuilder } from "../services/agreementService.js";
+// import { readModelServiceBuilder } from "../services/readModelService.js";
+
+// const readModelService = readModelServiceBuilder(config);
```

Comment:
leftover?

Reply:
Sorry, updated again to pass it as parameter.